lib/spack/spack/filesystem_view.py

```diff
@@ -96,6 +99,7 @@ def view_copy(src, dst, view, spec=None):
                 files=[dst],
                 prefixes=prefix_to_projection
             )
+        fp.set_permissions_by_spec(dst, spec)
```

Comment:
Is this still necessary after switching to `copy2` above?

Reply:
I haven't tested it. I don't think so but I do want to keep groups so something will have to be added in its place

Reply:
updated this.